### PR TITLE
add epoch logging example

### PR DESCRIPTION
**Pull request by danabens**
Add tracker example of logging epoch based metrics via Tracker.

### CODE REVIEW

File: sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+data
```
Comment (jkasiraj) [inline sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore:1]: minor: remove the .gitignore?

**Review by jkasiraj — APPROVED** [review]

File: sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore (hunk 1 above)
Comment (danabens) [reply sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore:1]: why? the notebook downloads a bunch of data, and this excludes it from git operations

Comment (jkasiraj) [reply sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore:1]: makes sense, it just didn't seem related to the other new changes so wasn't sure if it was intentional

**Review by jkasiraj — APPROVED** [review]

File: sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore (hunk 1 above)
Comment (jkasiraj) [reply sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore:1]: makes sense, it just didn't seem related to the other new changes so wasn't sure if it was intentional

**Pull request closed without merging**